src/test/java/org/folio/search/controller/SearchInstanceFilterIT.java

```diff
@@ -328,7 +328,7 @@ private static Stream<Arguments> facetQueriesProvider() {
       "tenantId", "shared");
     return Stream.of(
       arguments("id=*", allFacets, mapOf(
-        "shared", facet(facetItem("false", 3), facetItem("true", 2)),
+        "shared", facet(facetItem("false", 5)),
```

Comment:
Why are there no "true" values?
Test data should be changed in a scope of MSEARCH-562 in order to have them?

Reply:
Yes, we should implement some setup for consortia tests, where we will have more than 1 tenant enabled.

Reply:
We currently have 2 tenants setup for integration tests, see BaseIntegrationTest:enableTenant usage.
But possibly we lack 'shared' logic